tokens/blocket.se/deprecated.yml

```diff
@@ -0,0 +1,71 @@
+---
+token: maps
+
+color:
+  focused: aqua-400
+  background:
+    _: white
+    subtle: bluegray-50
+    interactive:
+      _: white
+      hover: bluegray-100
+      selected: bluegray-100
+  text:
+    _: gray-700
+    subtle: gray-500
+    placeholder: bluegray-300
+    inverted:
+      _: white
+      subtle: gray-50
+    link:
+      _: blue-600
+      hover: blue-800
+      visited: inherit
+
+s:
+  color:
+    background:
+      default: white
```

Comment:
Here I was also wondering if it would make sense to have a default value for the below listed reset styles that rely on new css vars, where the defaults would be the same as the css vars available at https://assets.finn.no/pkg/@warp-ds/tokens/1.0.0-alpha.92/finn-no.css:

```
*,
::before,
::after {
...
  border-color: var(--w-s-color-border-default, var(--w-s-color-border));
}

body {
...
  background-color: var(--w-s-color-background-default, var(--w-s-color-background));
}

body, :host {
...
  color: var(--w-s-color-text-default, var(--w-s-color-text)); 
}
```

This would help only in case where a podlet imports resets.css from the latest version of @warp-ds/css assets, while the layout is still using an old brand CSS for some reason.
Happy to discuss this before making any commits with this 😄

Reply:
Then I guess we will have to live with this duplication forever... as it will be hard to get our consumers to stop using classes like `s-bg-default`, `s-border-negative-subtle-default` and so on, when more and more adds them to production?

Reply:
Hm. You mean that when users see that `s-bg-default` still works (cause we still keep the token) they will continue using it instead of `s-bg`? Because my thought here was only to make sure the above examples from resets work regardless of layout's brand CSS version (meaning /tokens/v1/... vs. /css/<version>/tokens/...). But if it's a matter of making stuff break in prod to force users to update their styles, I would maybe try to inform those few teams that have their stuff in prod that they should first replace the classes/css vars to the new ones anyway. Then once we see that everyone has done that + there's no mention of the old brand CSS file in the finn code, we can remove the deprecated.yml and the above safeguards in resets.css. At least that's my idea of the backward compatibility at this stage :D

Reply:
I opened a PR with a suggested safeguard for reset.js here: https://github.com/warp-ds/css/pull/30 if you think having it makes sense 😊